clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/DocumentReadGroup.scala

```diff
@@ -2,15 +2,16 @@ package org.broadinstitute.clio.server.dataaccess.elasticsearch
 
 import java.time.OffsetDateTime
 
-case class DocumentReadGroup(analysisType: Option[String],
+case class DocumentReadGroup(flowcellBarcode: String,
+                             lane: Int,
+                             libraryName: String,
+                             location: String,
```

Comment:
I think Dan is right, but sealed trait does seem like overkill.
I suppose it is better than "magic" strings though.